indra/resources/update_resources.py

```diff
@@ -662,11 +664,13 @@ def _process_mirbase_file(lines):
         for element in group:
             if not element.startswith('DR'):
                 continue
-            db, identifier, name = [e.strip() for e in element[len('DR'):].lstrip().split(';')]
+            db, identifier, name = [e.strip() for e in \
+                                       element[len('DR'):].lstrip().split(';')]
             yield mirbase_id, mirbase_name, db, identifier, name.rstrip('.')
 
 
 if __name__ == '__main__':
+    """
```

Comment:
Hey, what's going on here?

Reply:
D'oh, commented stuff out for testing and then forgot to uncomment 🤦‍♂

Reply:
Fixed